components/feature/readerview/src/main/assets/extensions/readerview/readerview.js

```diff
@@ -70,7 +70,7 @@ class ReaderView {
     result.language = document.documentElement.lang;
 
     let article = Object.assign(
-      result,
+      result,r
```

Comment:
It looks like this is an artifact from the other git merge issues. :)

Reply:
I'm not sure how that got in there. Sorry!